doc/refman/RefMan-syn.tex

```diff
@@ -1322,6 +1322,57 @@ \subsubsection{\tt Print Scopes\comindex{Print Scopes}}
 %except that abbreviations are used for printing (unless the modifier
 %\verb=(only parsing)= is given) while syntactic definitions were not.
 
+\section{Numeral Notations
+\comindex{Numeral Notation}}
+
+Starting with version 8.8, the following command allows to customize
+the way numeral literals are parsed and printed:
+
+\begin{quote}
+{\tt Numeral Notation $ty$ $pa$ $pr$~:~{\scope}.}
+\end{quote}
+
+In the previous line, $ty$ should be the name of an inductive type,
+while $pa$ and $pr$ should be the names of two parsing and printing
+functions.
+The parsing function $pa$ should have one of the following types:
+\begin{itemize}
+\item {\tt Decimal.int -> $ty$}
+\item {\tt Decimal.int -> option $ty$}
+\item {\tt Decimal.uint -> $ty$}
+\item {\tt Decimal.uint -> option $ty$}
+\item {\tt Z -> $ty$}
+\item {\tt Z -> $ty$ option}
+\end{itemize}
+And the printing function $pr$ should have one of the following types:
+\begin{itemize}
+\item {\tt $ty$ -> Decimal.int}
+\item {\tt $ty$ -> option Decimal.int}
+\item {\tt $ty$ -> Decimal.uint}
+\item {\tt $ty$ -> option Decimal.uint}
+\item {\tt $ty$ -> Z}
+\item {\tt $ty$ -> option Z}
+\end{itemize}
+
+\begin{Variants}
+\item
+{\tt Numeral Notation $ty$ $pa$ $pr$~:~{\scope} (warning after $n$).}
+
+When a literal larger than $n$ is parsed, a warning message about
+possible Stack Overflow will be displayed.
```

Comment:
Should stack overflow be capitalized?

Reply:
I don't think so. When capitalized, it seems to refer to the Stack Overflow website.

Reply:
Done, thanks